src/components/existing-request/existing-request-display.vue

```diff
@@ -250,9 +250,15 @@ export default class ExistingRequestDisplay extends Mixins(
     })
   }
 
-  private get address () {
-    const fields = ['addrLine2', 'city', 'stateProvinceCd', 'countryCd', 'postalCd']
+  private get address (): string {
+    // FUTURE: delete this check as it hides an error that shouldn't happen
+    //         for now, report the error and don't crash
+    if (!this.nr.applicants) {
```

Comment:
As per the comment, `this.nr` and/or `this.nr.applicants` should never be null/undefined, but according to Sentry this has definitely happened.

So this is a work-around for now and hides the error, but I added a console.error() to report it in Sentry so we remember to fix/verify this later.